packages/core/src/TableSection/TableSection.styles.tsx

```diff
@@ -85,6 +85,12 @@ export const { staticClasses, useClasses } = createClasses("HvTableSection", {
     [`& .${paginationClasses.pageSizeOptions}`]: {
       left: theme.space.sm,
     },
+    [`& .${paginationClasses.pageSizeHeader}`]: {
+      border: "none",
```

Comment:
Shouldn't we also remove the border of the popup content? This looks strange...
![image](https://github.com/lumada-design/hv-uikit-react/assets/638946/5c334e0d-a731-42b9-b699-bbe2a7d71350)